src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,348 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: A summary of commands used to convert a raw mov file to an encrypted full HD file for web playback.
+
+{# wf_updated_on: 2017-06-05 #}
+{# wf_published_on: 2017-06-05 #}
+
+# Media Manipulation Cheat Sheet  {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+This document shows in order commands needed to get from a raw mov file to
+encrypted assets packaged for DASH or HLS. For the sake of having a goal to
+illustrate, I'm converting my source file to a bitrate of 8Mbs at a resolution
+of 1080p (1920 x 1980). Adjust these values as your needs dictate.
+ 
+Conversion is done with two applications:
+[Shaka Packager](https://github.com/google/shaka-packager) and
+[ffmpeg](https://ffmpeg.org/download.html). Although
+I've tried to show equivalent operations for all procedures, not all operations
+are possible in both applications.
+ 
+In many cases, commands may be combined in a single command line operation, and
+in actual use would be. For example, there's nothing preventing you from setting
+an output file's bitrate in the same operation as a file conversion. For this
+cheat sheet, I will show these operations (among others) as separate commands
+for the sake of clarity.
+ 
+Please let me know of useful additions or corrections. [Pull requests are welcome](https://github.com/google/WebFundamentals).
+
+## Display characteristics
+
+    packager input=myvideo.mp4 --dump_stream_info
+
+    ffmpeg -i myvideo.mp4
+
+This is technically an incorrect usage of ffmpeg. As such it prints an error
+message to the screen. It does however list information not available from the
+Shaka Packager command.
+
+## Demux (split audio and video)
+
+My preference would be to show file type conversion before diving into demuxing.
+It appears that Shaka Packager cannot do conversion without also demuxing.
+
+**mp4/Shaka Packager**
+ 
+    packager input=myvideo.mp4,stream=video,output=myvideo_video.mp4
+    packager input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+ 
+With Shaka Packager you can combine these.
+ 
+    packager \
+      input=myvideo.mp4,stream=video,output=myvideo_video.mp4 \
+      input=myvideo.mp4,stream=audio,output=myvideo_audio.m4a
+      
+**webm/Shaka Packager**
+
+    packager \
+      input=myvideo.webm,stream=video,output=myvideo_video.webm \
+      input=myvideo.webm,stream=audio,output=myvideo_audio.webm
+ 
+**mp4/ffmpeg**
+ 
+    ffmpeg -i myvideo.mp4 -vcodec copy -an myvideo_video.mp4
+    ffmpeg -i myvideo.mp4 -acodec copy -vn myvideo_audio.m4a
+    
+**webm/ffmpeg**
+ 
+    ffmpeg -i myvideo.webm -vcodec copy -an myvideo_video.webm
+    ffmpeg -i myvideo.webm -acodec copy -vn myvideo_audio.webm
+
+## Change characteristics
+
+### Bit rate
+
+For ffmpeg, I can do this while I'm converting to mp4 or webm.
```

Comment:
For all these, but especially for bit rate, it would be helpful to have a single phrase that describes what the example does - "split the video out from myvideo.mp4 into myvideo_video.mp4, then the audio into myvideo_audio.mp4", "Recode myvideo.mov into a myvideo.mp4 at an 8000000bps bitrate", etc.